pkg/git/utils.go

```diff
@@ -0,0 +1,48 @@
+package git
+
+import (
+	"strings"
+)
+
+// CheckForValidForkRemote checks if the remote URL is a valid remote fork for the upstream URL.
+func CheckForValidForkRemote(upstreamURL, remoteURL, repo string) bool {
+	urlPart1, urlPart2 := extractCommonParts(upstreamURL, remoteURL)
+	if urlPart1 == "https://github.com" && urlPart2 == repo {
+		return true
+	}
+
+	return false
```

Comment:
We can simplify the entire body of this funtion:

```go
return urlPart1 == "https://github.com" && urlPart2 == repo
```

Reply:
Keen eyes, fixing.